commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitControllers.scala

```diff
@@ -118,7 +118,11 @@ trait PurePursuitControllers extends UnicycleCoreControllers {
       )
     }
 
-    val headingToTarget = position.zip(lookAheadPoint).map(p => headingToPoint(p._1, p._2))
+    val headingToTarget = position.zip(lookAheadPoint).map{p =>
+      val seg = Segment(p._1, p._2)
```

Comment:
This is unused?

Reply:
This is also here for debug purposes. It will be removed in the final version of the pr